### PR TITLE
PRIME-2701 Expand Practitioner Table

### PR DESCRIPTION
**Pull request by bergomi02**
expand practitioner table

### CODE REVIEW

File: prime-dotnet-webapi/Models/Practitioner.cs [hunk 1]
```diff
@@ -11,15 +11,14 @@ public class Practitioner : BaseAuditable
...
         public string PracRefId { get; set; }
-
         public string FirstName { get; set; }
-
+        public string MiddleInitial { get; set; }
         public string LastName { get; set; }
-
+        public DateTime DateofBirth { get; set; }
+        public string Status { get; set; }
+        public DateTimeOffset EffectiveDate { get; set; }
```
Comment (neophyte57) [inline prime-dotnet-webapi/Models/Practitioner.cs:21]: Better as `DateTimeOffset?`

File: prime-dotnet-webapi/Models/Practitioner.cs [hunk 2]
```diff
@@ -11,15 +11,14 @@ public class Practitioner : BaseAuditable
...
         public string CollegeId { get; set; }
-
         public string PracRefId { get; set; }
-
         public string FirstName { get; set; }
-
+        public string MiddleInitial { get; set; }
         public string LastName { get; set; }
-
+        public DateTime DateofBirth { get; set; }
```
Comment (neophyte57) [inline prime-dotnet-webapi/Models/Practitioner.cs:19]: Better as `DateTime?`

File: prime-dotnet-webapi/Models/Practitioner.cs (hunk 1 above)
Comment (bergomi02) [reply prime-dotnet-webapi/Models/Practitioner.cs]: Good idea. It is updated.

File: prime-dotnet-webapi/Models/Practitioner.cs (hunk 2 above)
Comment (bergomi02) [reply prime-dotnet-webapi/Models/Practitioner.cs]: It is updated.

**Review by neophyte57 — APPROVED** [review]
LGTM!

**Pull request merged**